.azuredevops/rocm-ci.yml

```
@@ -0,0 +1,42 @@
+resources:
+  repositories:
+  - repository: pipelines_repo
+    type: github
+    endpoint: ROCm
+    name: ROCm/ROCm
+
+variables:
+- group: common
+- template: /.azuredevops/variables-global.yml@pipelines_repo
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - develop
+    - mainline
+  paths:
+    exclude:
+    - .github
+    - docs
+    - '.*.y*ml'
```

Since this intention here is to exclude YAML files
```suggestion
    - '.*.yaml'
    - '.*.yml'
```